<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Lakshmi" name="Wff" tests="1" failures="1" timestamp="10 Jul 2015 11:23:30 GMT" time="60.094" errors="0">
  <testcase name="testLeaplogin" time="60.052" classname="leaplogin.BrowsersTestMain">
    <failure type="java.lang.AssertionError" message="timeout">
      <![CDATA[java.lang.AssertionError: timeout
	at leaplogin.BrowsersTestMain.testLeaplogin(BrowsersTestMain.java:54)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- testLeaplogin -->
</testsuite> <!-- Wff -->
